pkg/accounting/accounting.go

```diff
@@ -365,7 +365,7 @@ func (c *creditAction) Apply() error {
 	c.accounting.metrics.CreditEventsCount.Inc()
 
 	if c.price.Cmp(c.accountingPeer.reservedBalance) > 0 {
-		c.accounting.logger.Error(ErrOverRelease, "for peer %v")
+		c.accounting.logger.Error(ErrOverRelease, ErrOverRelease.Error(), "peer", c.peer)
```

Comment:
ErrOverRelease.Error() would cause the error string to be printed twice

Reply:
Correct, this `ErrOverRelease.Error()` should be some meaningful log message.

Reply:
you have not resolved this :(